src/libraries/Common/src/System/Runtime/InteropServices/ComEventsMethod.cs

```diff
@@ -156,46 +157,40 @@ public bool Empty
         {
             get
             {
-                lock (_delegateWrappers)
-                {
-                    return _delegateWrappers.Count == 0;
-                }
+                DelegateWrapper[] wrappers = _delegateWrappers;
+                return wrappers.Length == 0;
```

Comment:
Nit: Could just be `get => _delegateWrappers.Length == 0;`, but if you prefer having the local, ok